cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,52 @@
-describe('', () => {
-  before(() => {
+describe('Article flow', () => {
+  let user;
+  let articleData;
 
+  beforeEach(() => {
+    cy.visit('');
```

Comment:
It's better to provide an actual URL instead of an empty string in `cy.visit()`. If you intended to visit the base URL, you can omit the argument altogether.